hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1151,7 +1151,13 @@
   regressions: [
     {
       name: smoke
-      tests: ["chip_sw_uart_tx_rx", "chip_plic_all_irqs"]
+      tests: ["xbar_chip_smoke",
+              "chip_sw_uart_tx_rx",
+              "chip_plic_all_irqs",
+              "chip_sw_example_flash",
+              "chip_sw_example_rom",
+              "chip_sw_example_manufacturer",
+              "chip_sw_uart_smoketest_signed"]
```

Comment:
Maybe you want something even simpler, i.e. the [ROM boot success test](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/dv/chip_rom_tests.hjson;drc=cc8c8916cf8c7fbb47fcb376788d2733c8bc448a;l=12), as it completes faster. ~100 min (see #14382) vs. ~75min (see #14383). 

Also, just an FYI, running this test will increase the private CI time substantially. Right now the smoke test regression takes around ~30-40 min on average across PRs to finish. A **_single_** test with the production ROM takes on the order of ~75+ min (which would over double the private CI runtime). The [timeout](https://github.com/lowRISC/opentitan-private-ci/blob/c9fbbf910bc60792975915be28a2536811055ec5/jobs.yml#L25) for this private CI job is 120 min, so this will start to push the limit depending on how the job is scheduled. Moreover, right now the CI critical path is the FPGA tests due to Jake queueing. However, this is currently being addressed by swapping out the machine that is connected to the FPGA so these dv sims could end up being the critical path. 

We may want to hold off running ROM tests at all in presubmit. Alternatively, I'm also OK with giving it a try and if its a CI burden, then we can roll it back quickly.

Reply:
wow is a production rom test supposed to take that long? Is it doing software verification? I thought the last test I tried using hardware verification went quite a bit faster.

Reply:
Yes, it looks like the default OTP image (in RMA LC state) that is used for all chip-level tests has the [`CREATOR_SW_CFG_USE_SW_RSA_VERIFY` field](https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/otp_ctrl/data/otp_ctrl_img_rma.hjson;drc=b4d0de8f261f6397c4af74d33b0dc7afae73e224;l=28) set to do signature verification in SW. Should we change this to have it default to using HW?
CC: @alphan

Reply:
BTW, I am running the tests with the OTP configured to use HW for verification, I will post the results here when the tests complete.

Reply:
Sure, I just picked something at random and thought I will let the CI check tell me how it impacts the runtime. Didn't expect comments on this PR already :)
I am good with not running any mask ROM tests in CI.

Reply:
yeah i think for our testing purposes we probably want to do HW verification.
It might make sense to have 1 test that does software just to make sure it works, but probably not something we want to apply to all tess.

I do recall changing it to HW verification some time ago.. maybe it got changed back or I missed one of the configuration files.

Reply:
> Yes, it looks like the default OTP image (in RMA LC state) that is used for all chip-level tests has the [`CREATOR_SW_CFG_USE_SW_RSA_VERIFY` field](https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/otp_ctrl/data/otp_ctrl_img_rma.hjson;drc=b4d0de8f261f6397c4af74d33b0dc7afae73e224;l=28) set to do signature verification in SW. Should we change this to have it default to using HW? CC: @alphan

I'm not sure how _official_ these definitions are but I think we want the initial RMA configuration to choose the SW implementation, at least for ES, until we confirm that OTBN works as expected. We can switch to OTBN once we are confident. For tests, I think it's fine to do whatever but we should probably cover actual configurations. WDYT?

Reply:
for tests, we probably should consider picking a specific test for software... otherwise the simulation time will seriously be out of this world. 

If you prefer, we can keep the images as software (so fpga will always be softawre), and in DV we can override the symbol to hardware, that will probably give us the best of both worlds.

Reply:
> for tests, we probably should consider picking a specific test for software... otherwise the simulation time will seriously be out of this world.
> 
> If you prefer, we can keep the images as software (so fpga will always be softawre), and in DV we can override the symbol to hardware, that will probably give us the best of both worlds.

SGTM! Thanks @tjaychen!